### PR TITLE
Custom domain patches

### PR DESCRIPTION
**Pull request by philschanely**
## Description

This PR makes a few small non-breaking patches for tables, labels, and cards in React for work Manage is doing in Custom Domains.

### CODE REVIEW

**Review by dangkevin — APPROVED** [review]
looks good to me!

**Review by kimjaej — APPROVED** [review]
LGTM 👍

File: packages/sage-react/lib/Label/configs.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ export const LABEL_COLORS = {
   INFO: 'info',
   LOCKED: 'locked',
   PUBLISHED: 'published',
+  SUCCESS: 'published', // dupe alias for common usage or other contexts
```
Comment (joedakroub) [inline packages/sage-react/lib/Label/configs.js:7]: Nice 👍

**Review by joedakroub — APPROVED** [review]
No concerns here. Thanks Phil! 👍

**Pull request merged**